### ISSUE TITLE
question about compute_discriminator_loss 

### ISSUE THREAD

**Issue by xiaohaipeng** [NVlabs/SPADE#130]
when compute_discriminator_loss, why set requires_grad=True after .detach()?
`fake_image = fake_image.detach()
 fake_image.requires_grad_()`

**Comment by 15534081591** [comment]
请问您知道什么原因了吗
